upgrade pynac to 0.7.7

saraedum opened the pull request.

saraedum committed:
upgrade pynac to 0.7.7

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

isuruf reviewed (commented):
We should wait for the new singular version and pin to that version exactly.

saraedum committed:
Merge branch 'master' of https://github.com/conda-forge/pynac-feedstock into pynac077

saraedum committed:
Reset build number

saraedum committed:
pin to Singular's patchlevel release

We need to pin even the patchlevel as Singular has changed the version
numbering of their shared libraries with that patch release.

saraedum commented:
Ok. I added the pinning here.

isuruf commented:
Python3 fails. The errors were fixed in 0.7.8. Does sage work with 0.7.8?

isuruf committed:
Skip python 3.x for 0.7.7

isuruf committed:
MNT: Re-rendered with conda-smithy 2.3.2

isuruf commented:
Thanks
